Profile/EducationProfile.cs

```diff
@@ -0,0 +1,62 @@
+﻿using MarsQA_1.Helpers;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System.Threading;
+
+namespace MarsQA_1.SpecflowPages.Pages
+{
+    public class EducationProfile
+    {
+        public string Country { get; set; }
+        public string University { get; set; }
+        public string Title { get; set; }
+        public string Degree { get; set; }
+        public int YearOfGraduation { get; set; }
+        public string EducationMessage { get; set; }
+
+        IWebElement EducationTab => Driver.driver.FindElement(By.XPath("(//A[text()='Education'])[1]"));
+        IWebElement AddNewButton => Driver.driver.FindElement(By.XPath("(//div[text()='Add New'])[3]"));
+        IWebElement UniversityName => Driver.driver.FindElement(By.XPath("//INPUT[@placeholder='College/University Name']"));
+        IWebElement CountryName => Driver.driver.FindElement(By.XPath("//SELECT[@class='ui dropdown'][@name='country']"));
+        IWebElement TitleDropDown => Driver.driver.FindElement(By.XPath("//SELECT[@class='ui dropdown'][@name='title']"));
+        IWebElement DegreeName => Driver.driver.FindElement(By.XPath("//INPUT[@placeholder='Degree']"));
+        IWebElement GraduationYear => Driver.driver.FindElement(By.XPath("//SELECT[@class='ui dropdown'][@name='yearOfGraduation']"));
+        IWebElement AddButton => Driver.driver.FindElement(By.XPath("(//input[@type='button'][@value='Add'])"));
+        string VerifyAction = "//div[@class='ns-box-inner']";
+        IWebElement AlertBoxMessage => Driver.driver.FindElement(By.XPath(VerifyAction));
+
+        public void AddEducation()

```

Comment:
Please pass the data from specflow feature file, then to the Steps class, then into the methods in this class instead of using global properties which you declared above